ncimonitor/UsageDataset.py

```diff
@@ -314,3 +314,39 @@ def getsystemstorage(self, systemname, storagepoint, year, quarter):
         if q is None:
             return (None,None)
         return float(q['grant']),float(q['igrant'])
+
+
+    def top_usage(self, storagepoint, count=10, date=None, measure='size'):
+        """
+        Return the top ``count`` users according to ``measure`` (either 'size'
+        or 'inodes') on ``storagepoint`` at ``date``
+
+        Due to the NCI scanning process ``date`` should be at least one day in
+        the past
+
+        Returns columns (measure, username, fullname)
+        """
+
+        if storagepoint not in ['short', 'gdata']:
+            raise Exception(f"Unexpected storagepoint '{storagepoint}'")
```

Comment:
Change to format or drop python 2 support. Your choice. I'm happy with the latter. We don't intend to use it with python 2 in the future.